openedx/core/djangoapps/safe_endpoints/middleware.py

```diff
@@ -0,0 +1,45 @@
+"""
+Middleware to ensure safe endpoints.
+"""
+import logging
+
+from rest_framework_jwt.authentication import BaseJSONWebTokenAuthentication
+
+
+log = logging.getLogger(__name__)
+
+
+class EnsureJWTAuthSettingsMiddleware(object):
+    """
+    Django middleware object that ensures the proper Permission classes
+    are set on all endpoints that use JWTAuthentication.
+    """
+    required_permission_classes = ()
```

Comment:
I like the use of an `_` prefix to designate code that is meant to be private.

Reply:
Me too.  Updated.